parachains/integration-tests/e2e/collectives/collectives-polkadot/1_alliance/1_init_alliance.yml

```diff
@@ -104,6 +104,7 @@ tests:
                 - name: sudo.Sudid
                   result: { sudoResult: Ok }
                 - name: xcmPallet.Sent
+                  result: { origin: { parents: 0, interior: Here }, destination: { parents: 0, interior: { X1: { Parachain: *coll_para_id }}}}
```

Comment:
maybe we should pull this up into a variable so we specify it only once? I think we can do that here.

Reply:
I think `xcmPallet.Sent`/`polkadotXcm.Sent` event attributes do not provide useful information and can be removed/ignored. It is just reflecting what has been sent which unequivocally will match with what was sent.

Just expecting `xcmPallet.Sent`/`polkadotXcm.Sent` without attributes would be enough

Reply:
it just helps to ensure that you asserting the event from the extrinsic above, not something from prev test for example